src/peripheral/dwt.rs

```diff
@@ -70,6 +70,34 @@ impl DWT {
         unsafe { self.ctrl.modify(|r| r | 1) }
     }
 
+    /// Whether to enable exception tracing
+    // TODO find out if this is supported om armv6m
+    #[inline]
+    pub fn enable_exception_tracing(&mut self, bit: bool) {
```

Comment:
Should an enable argument be taken or should two `enable_exception_tracing`, `disable_exception_tracing` function be exposed instead, alike the API for DCB?

Reply:
I think either have `set_exception_tracing(&mut self, enabled: bool)` _and/or_ `enable_exception_tracing(&mut self)`+`disable_exception_tracing(&mut self)`, but not `enable_exception_tracing(&mut self, bit: bool)`.

Reply:
Refactored into disable/enable functions.